release-team/role-handbooks/enhancements/README.md

```diff
@@ -84,65 +95,87 @@ It is important that this process be followed and documentation remain up-to-dat
 - Find [Issues labeled `tracked/yes`](https://github.com/kubernetes/enhancements/issues?q=is%3Aopen+is%3Aissue+label%3Atracked%2Fyes) and change to `tracked/no` until the Enhancement is ready to be tracked for the upcoming release.
 - Close previous milestone by ensuring that there are no open issues/PRs in that milestone.
+- Gather Shadows to have them read this handbook and give expectations on what the process looks like and their particular role. If possible, try to schedule a call with the shadows to get them accustomed to the team. This helps as a great team building exercise.
 
 #### Pre-Enhancements Freeze (Week 1)
 
-- Gather Shadows to have them read this handbook and give expectations on what the process looks like and their particular role. If possible, try to schedule a call with the shadows to get them accustomed to the team. This helps as a great team building exercise.
 - Send an email to the [Kubernetes-Dev](https://groups.google.com/forum/#!forum/kubernetes-dev) mailing list and a message to #chairs-and-techleads slack channel with a call for enhancements and how to opt-in to the release.
 - Verify issues have k/k PRs associated so they can be referenced and easily tracked. This is going to be critical come Enhancement Freeze and Code Freeze to see the status of the code.
 - Work with the Release Lead to introduce yourself, talk about release information, and relay information about opting into the release with SIG Leads.
 - Weekly Release meetings require updates of current status. Use the `Feature stats` tab to update the release team on counts of enhancements in good and bad progress.
-- Start reminding Issue owners that KEPs are required for each enhancement and that KEPs must be in an implementable state by Enhancement Freeze.
 
 #### Collection Monitoring and Triage (Week 2-3)
 
-- Move data from `KEP Collection` into the `data` tab to be tracked officially for the release
-- Stay on top of comments in issues when owners respond. Apply correct labels, milestone information, update their status in the sheet if necessary.
+- Monitor the `KEP Collection` tab regularly for new additions as SIGs opt-in KEPs into the release.
+- Move data from `KEP Collection` into the `data` tab to be tracked officially for the release.
+- Manually add the KEP issue number into the `Enhancements` tab and verify that the KEP is now included.
+  - Entering the KEP issue number will auto populate the Enhancement Name, SIG, KEP Assignee, Proposal, and KEP State.
+  - Fill out rest of the column manually based on current status of the KEP.
+- For the opted in issues, apply the correct milestone, `tracked/yes` label, and `stage/xxx` label.
```

Comment:
Should we enumerate the possible stage labels?
```suggestion
- For the opted in issues apply the correct milestone, `tracked/yes` label, and correct stage label (`stage/alpha`, `stage/beta`, or `stage/stable`) for the stage the enhancement is graduating to in this release.
```

Reply:
As an extension to the above comment, do we need a `stage/deprecation` label as well? We might see more deprecations in the near future due to recent changes in policy like the prevent perma-beta policy.

Reply:
@JamesLaverack @palnabarun Can we leave it generic so that not all stages (currents and future) need to be added to the handbook?

As for the `stage/deprecation` label, that's a great idea but since that's nonexistent right now, I think we should keep that separate from this PR

Reply:
we can bring up the `stage/deprecation` in the next enhancements meeting and get that label added.

Reply:
> Can we leave it generic so that not all stages (currents and future) need to be added to the handbook?

Sure. We can keep the statement as it was intended to be. 👍🏽

Reply:
+1 from me on discussing this in the enhancements meeting.

Reply:
If these are subject to change, as it sounds like they might soon, then +1 for leaving it generic.